commands/deploy.go

```diff
@@ -159,6 +159,7 @@ func runDeploy(cmd *cobra.Command, args []string) {
 			os.Exit(1)
 		}
 
+		fmt.Printf("GOING IN: %s, %s, %s, %s, %s\n", fprocess, gateway, functionName, image, language)
```

Comment:
Is this a stray debug line?

Reply:
Ah I saw it come up in the git diff, but it wasn't there in the editor pane, assumed VSCode was playing up - but yep, will nuke.